rust/connlib/snownet/src/allocation.rs

```diff
@@ -498,6 +508,10 @@ impl Allocation {
         })
     }
 
+    pub fn refresh(&mut self) {
```

Comment:
This is still unused for now because I need https://github.com/firezone/firezone/pull/3590 to merge first.